### PR TITLE
feat: Add Clean all command

### PR DESCRIPTION
**Pull request by jkciesluk**
Connected to https://github.com/scalameta/metals/pull/5060

### CODE REVIEW

File: packages/metals-vscode/package.json [hunk 1]
```diff
@@ -484,6 +484,11 @@
         "category": "Metals",
         "title": "Zip reports"
       },
+      {
+        "command": "metals.reset-workspace",
+        "category": "Metals",
+        "title": "Reset workspace"
```
Comment (kpodsiad) [inline packages/metals-vscode/package.json:490]: nitpick: is `Reset workspace` descriptive enough for people? Maybe sth more similar to what Intellij has will be better? Maybe sth more verbose like "Clear caches and reimport project"?

Comment (jkciesluk) [reply packages/metals-vscode/package.json]: Good idea, I changed it

**Comment by jkciesluk** [comment]
@tgodzik can we merge?

**Review by tgodzik — APPROVED** [review]
Sure!

**Pull request merged**